audb/core/load_to.py

```diff
@@ -322,15 +308,30 @@ def load_to(
     )
     if update:
         if only_metadata:
-            files = deps.attachment_files + deps.tables
+            files = deps.attachments + deps.tables
         else:
             files = deps.files
         for file in files:
             full_file = os.path.join(db_root, file)
             if os.path.exists(full_file):
-                checksum = audbackend.md5(full_file)
+                checksum = utils.md5(full_file)
```

Comment:
We need to call a custom md5 function in case `full_file` is a folder. In that case the checksum is created from all files in the folder (including the file structure).